app/src/main/java/org/fossasia/phimpme/sharedimgur/ImgurOuthActivity.java

```diff
@@ -0,0 +1,162 @@
+package org.fossasia.phimpme.sharedimgur;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.text.format.DateUtils;
+import android.util.Log;
+import android.view.View;
+import android.webkit.CookieManager;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.TextView;
+
+import org.fossasia.phimpme.R;
+import org.fossasia.phimpme.utilities.BasicCallBack;
+import org.fossasia.phimpme.utilities.Constants;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+import static org.fossasia.phimpme.utilities.Constants.IMGUR_LOGIN_URL;
+
+/**
+ * Created by manuja on 16/7/17.
+ */
+
+public class ImgurOuthActivity extends Activity {
```

Comment:
@mohitmanuja rename the class name either to `ImgurOAuthActivity` or `ImgurAuthActivity`